apps/dashboard/Gemfile

```diff
@@ -28,7 +28,8 @@ end
 group :test do
   gem 'capybara'
 
-  gem "selenium-webdriver"
+  # lock selenium-webdriver to this version until we upgrade to Rails 8.
+  gem "selenium-webdriver", "4.26.0"
```

Comment:
Higher versions aren't compatible with Ruby 3.1 and 3.2 and have errors with our current Rails version, so we have to lock it until we can upgrade Rails and/or ruby versions.